index.Rmd

```diff
@@ -10,10 +10,13 @@ description: "This book provides a lab exercise that introduces students to the
 favicon: assets/GDSCN_style/gdscn_favicon.ico
 ---
 
-# About this Course {-}
+# About {-}
```

Comment:
Note that this exercise fits well into a 3 hour lab (and that the Instructor Guide summarizes how that time budget is spent across the activities/exercises)

Reply:
@cutsort added a new "Format" section - what do you think?